dev-docs/test-3rd-party-apps-manually.md

```diff
@@ -3,7 +3,7 @@
 ### 1. Create a VM and SSH into it
 
 ```
-gcloud compute instances create --zone us-central1-a --image-project debian-cloud --image-family debian-10 test-app
+gcloud compute instances create --zone us-central1-a --image-project debian-cloud --image-family debian-12 test-app
```

Comment:
Should we not be testing on `debian-11` (the oldest available) by default? And then, why not `ubuntu-os-cloud:ubuntu-2004-lts` (which is actually older)?

Reply:
[e18feca](https://github.com/GoogleCloudPlatform/ops-agent/pull/1752/commits/e18feca17e427d43c5d2ed801276a535b2105733)